src/main/java/uk/gov/justice/hmpps/datacompliance/repository/jpa/model/retention/ManualRetention.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.justice.hmpps.datacompliance.repository.jpa.model.retention;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.Length;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(of = {"manualRetentionId"})
```

Comment:
Not saying don't do this.  Just be aware (if not already) the impact of doing it in unit tests around equality checking.